### PR TITLE
Documenting the Clear Cart Tag & Remove Line Item Tag

### PR DESCRIPTION
**Pull request by JHarrison89**
This commit adds documentation for the Clear Cart tag and Remove Item Tag, including liquid tag input examples and the rendered result.

### CODE REVIEW

File: docs/reference/tags/form_tags/clear_cart/index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+On button click, the Clear Cart Tag will redirect to the cart page by default.
+To specify the redirect location, pass a URL as the value of the return_to: param.
+
+The following example redirects to the homepage. 
+
+##### input
+{% raw %}
+```liquid
+{% form "clear_cart", return_to: 'http://www.creator-site.com/home' %}
+	 <input type="submit" value="clear" class="cart__clear {{id}}__clear" />
```
Comment (geoffreymm) [inline docs/reference/tags/form_tags/clear_cart/index.md:40]: Given those classes aren't rendered by the tag, and aren't important for this to work, I'd remove it.

File: docs/reference/tags/form_tags/clear_cart/index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+has_children: false
+---
+
+The Clear Cart Tag renders a button that clears the user's cart. 
+
+##### input
+{% raw %}
+```liquid
+{% form "clear_cart", return_to: config.clear_cart_link.url %}
+	 <input type="submit" value="clear" class="cart__clear {{id}}__clear" />
```
Comment (geoffreymm) [inline docs/reference/tags/form_tags/clear_cart/index.md:14]: Given those classes aren't rendered by the tag, and aren't important for this to work, I'd remove it.

File: docs/reference/tags/form_tags/clear_cart/index.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+parent: Tags
+has_children: false
+---
+
+The Clear Cart Tag renders a button that clears the user's cart. 
+
+##### input
+{% raw %}
+```liquid
+{% form "clear_cart", return_to: config.clear_cart_link.url %}
```
Comment (geoffreymm) [inline docs/reference/tags/form_tags/clear_cart/index.md:13]: The return_to is optional, we can add it to the section below where we described the optional param.

File: docs/reference/tags/form_tags/clear_cart/index.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+##### redirect
+On button click, the Clear Cart Tag will redirect to the cart page by default.
+To specify the redirect location, pass a URL as the value of the return_to: param.
+
+The following example redirects to the homepage. 
+
+##### input
+{% raw %}
+```liquid
+{% form "clear_cart", return_to: 'http://www.creator-site.com/home' %}
```
Comment (geoffreymm) [inline docs/reference/tags/form_tags/clear_cart/index.md:39]: I'd use a relative `path`, instead.

**Review by geoffreymm — COMMENTED** [review]
Looking nice! I think we can simplify a bit more removing the classes defined, they aren't important for the example.

We can follow the examples @paulina-a described on the comment:
https://www.notion.so/Create-Line-Item-Form-Tag-3349351d4d8247ef8beb1b16080692b6?d=bf09619f0d5342da8d57f0f385b6f844

**Review by geoffreymm — COMMENTED** [review]
Looks like the commit history is a bit confusing 🤔 maybe worth separate on different commits.

**Comment by JHarrison89** [comment]
[Missing tags documentation](https://trello.com/c/QNU4wV3t/263-missing-tags-documentation)

**Pull request closed without merging**